README.md

```diff
@@ -464,6 +607,34 @@ Populated after `ready` has been emitted. Will be `0` before the event.
 
 How much padding is applied to each block of this core? Will be `0` unless block encryption is enabled.
 
+#### `core.peers`
+
+Array of current peers the core is replicating with.
+
+#### `await core.setEncryption(encryption)`
+
+Set the encryption, which should satisfy the [HypercoreEncryption](https://github.com/holepunchto/hypercore-encryption) interface.
+
+#### `await core.setEncryptionKey(key, [opts])`
+
+Set the encryption key.
+
+`opts` includes:
+
+```
+{
+  block: false, // Whether the key is for block encryption
+}
+```
+
+#### `core.setKeyPair(keyPair)`
+
+Update the core's `keyPair`.
+
+#### `core.setActive(active)`
+
+Set the core to be active or not. A core is considered 'active' if it should linger to download blocks from peers.
```

Comment:
Note that when users call `setActive(true), they have to call `setActive(false)` when they're done, otherwise the book keeping breaks.

Reply:
Very true. Thanks! Added.